### DISCUSSION TITLE
New addition: AVR-DA-S family

### DISCUSSION THREAD

**Discussion by askn37** [avrdudes/avrdude#1885] [Ideas]
Details are still scarce, but it appears that new definitions have been added to the AT-Pack.

https://packs.download.microchip.com

> Microchip AVR-Dx Series Device Support (2.6.303)
> 2.6.303 (2024-08-01) | - Added AVR128DA28S, AVR128DA32S, AVR128DA48S, and AVR128DA64S. - Fixed spelling of ADC_START_EVENT_TTRIGGER to ADC_START_EVENT_TRIGGER.

**Comment by MCUdude** [comment]
It looks like they are very similar to the older DA series, but with a different device signature and different fuse settings. I think it's best if @stefanrueger adds support for these chips by utilizing his atdf scraper script

**Comment by mcuee** [comment]
The support will be included in the upcoming avrdude 8.1 release.
https://github.com/avrdudes/avrdude/blob/main/NEWS
